### DISCUSSION TITLE
[Projects beta] Limits on number of entries - New GitHub projects

### DISCUSSION THREAD

**Discussion by schalkneethling** [community/community#14798] [Projects and Issues]
I started using the bulk addition of issues and pull requests to projects(thank you for this!). It seems I have hit a limit with the number of items a project board can contain. When I try to add more items to our [project board here](https://github.com/orgs/mdn/projects/15/views/1), I get a `500 internal server error` in the console and the message, "something went wrong" in the UI.

![Screenshot of 500 server error in console when trying to add multiple issues to project](https://user-images.githubusercontent.com/10350960/163016063-370bfc71-a954-4d65-bd5d-ead4d4c4827c.png)

![Screenshot of code showing location where the error is thrown](https://user-images.githubusercontent.com/10350960/163016240-c633ae04-1011-403b-bd42-6d3739a3db33.png)

I also get that when trying to add an individual issue or PR, although here, it silently fails.

**Comment by schalkneethling** [comment]
Here is some additional info with regards to the error:

```
XHRGEThttps://github.com/_jobs/8dd6e7c8-0133-4642-8bf1-4ef702df8987
[HTTP/2 500 Internal Server Error 310ms]

	
job	Object { id: "8dd6e7c8-0133-4642-8bf1-4ef702df8987", state: "error", ttl: 604800 }
id	"8dd6e7c8-0133-4642-8bf1-4ef702df8987"
state	"error"
ttl	604800
```

**Comment by wbobeirne** [comment]
I also encountered this. It silently fails if you try to assign an issue to a project (No error, just 200s but doesn't assign to the project.) Very confusing until I tried to do it from the projects page!

**Reply by schalkneethling** [reply]
The best workaround for this at the moment is to periodically archive all the cards in your "Done" column, for example. The items that go to the archive do not count toward this limit.
